### PR TITLE
Add onLongPress to ListItem

### PR DESCRIPTION
**Pull request by nodece**
Add onLongPress to ListItem.

### CODE REVIEW

File: components/list/ListItem.tsx [hunk 1]
```diff
@@ -11,6 +11,8 @@ export interface ListItemProps
   onPress?: () => void;
   onPressIn?: () => void;
   onPressOut?: () => void;
+  delayLongPress?: number;
+  onLongPress?: () => void;
```
Comment (bang88) [inline components/list/ListItem.tsx]: onLongPress?: (event: GestureResponderEvent) => void;

Comment (nodece) [reply components/list/ListItem.tsx]: I think should amend ListItemProps.

```
onLongPress?: (event: GestureResponderEvent) => void;
onPress?: (event: GestureResponderEvent) => void;
onPressIn?: (event: GestureResponderEvent) => void;
onPressOut?: (event: GestureResponderEvent) => void;
```

Comment (nodece) [reply components/list/ListItem.tsx]: @bang88

Comment (nodece) [reply components/list/ListItem.tsx]: @bang88 Updated.

**Pull request merged**